### ISSUE TITLE
Blank screen after sending mail with attachment

### ISSUE THREAD

**Issue by fridzema** [beyondcode/helo-community#13]
After sending 1 email with a pdf attached (5 mb) and a few download url's in the message content, which maybe the linkchecker has difficulties with this?
After quitting and a full system reboot it is now a blank screen.
<img width="993" alt="Screen Shot 2020-04-23 at 16 07 36" src="https://user-images.githubusercontent.com/8180660/80108935-308a5600-857d-11ea-8f18-b1b1e41d689b.png">

Is there anyway i can reset the app?

Also an idea popped up yesterday working with the app, should be nice to have a delete all button or something.

**Comment by fridzema** [comment]
This cleared the package data for me!
```bash
rm -rf ~/Library/Cache/HELO
rm -rf ~/Library/Saved\ Application\ State/HELO
rm -rf ~/Library/Application\ Support/HELO
```

**Comment by mpociot** [comment]
Thank you for reporting this, I’ll look into this tomorrow

**Comment by mpociot** [comment]
This is now fixed in version 1.1.0
